logger test fails if ipython is not installed

```
test.test_logger_test TextLoggerExceptionInClassFixtureTestCase.test_class_teardown_raises_after_test_raises ... fail: test.test_logger_test TextLoggerExceptionInClassFixtureTestCase.test_class_teardown_raises_after_test_raises
Traceback (most recent call last):
  File "./test/test_logger_test.py", line 222, in test_class_teardown_raises_after_test_raises
    assert_in('FakeTestException', test1_raises_result['exception_info_pretty'])
AssertionError: assertion failed: expected 'FakeTestException' in ['Traceback (most recent call last):\n', '  File "./test/test_logger_test.py", line 214, in test1_raises\n    raise FakeTestException(\'I raise before class_teardown raises\')\n', 'FakeTestException: I raise before class_teardown raises\n']

FAIL in 0.01s
```

--- a/test/test_logger_test.py
+++ b/test/test_logger_test.py
@@ -219,8 +219,8 @@ class TextLoggerExceptionInClassFixtureTestCase(TextLoggerBaseTestCase):
             assert_equal(len(self.logger.results), 3)
             test1_raises_result = self.logger.results[0]
             class_teardown_result = self.logger.results[-1]
-            assert_in('FakeTestException', test1_raises_result['exception_info_pretty'])
-            assert_in('FakeClassFixtureException', class_teardown_result['exception_info_pretty'])
+            assert_in('FakeTestException', str(test1_raises_result['exception_info_pretty']))
+            assert_in('FakeClassFixtureException', str(class_teardown_result['exception_info_pretty']))
 
 
 if __name__ == '__main__':

